package.json

```diff
@@ -54,5 +54,28 @@
     "trailingComma": "es5",
     "singleQuote": true,
     "printWidth": 120
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.0.3",
+    "@typechain/ethers-v5": "^10.0.0",
+    "@typechain/hardhat": "^6.0.0",
+    "@types/chai": "^4.3.1",
```

Comment:
Do we need both `"@types/chai": "^4.3.1",` and `"chai": "^4.3.6",`?

Reply:
@types/chai adds in the types for Chai so that any typescript you write can be type checked. If we're manually importing or using Chai, we should add this. If it's only included transitively we can skip.